selfdrive/thermald/thermald.py

```diff
@@ -406,6 +407,8 @@ def thermald_thread():
     power_monitor.calculate(peripheralState, startup_conditions["ignition"])
     msg.deviceState.offroadPowerUsageUwh = power_monitor.get_power_used()
     msg.deviceState.carBatteryCapacityUwh = max(0, power_monitor.get_car_battery_capacity())
+    current_power_draw = HARDWARE.get_current_power_draw()
```

Comment:
Think the linter is unhappy about the implementation of this function in `selfdrive/hardware/eon/hardware.py` that returns None, but should probably just return 0.

But think this warning is a bit lame, since it doesn't care that you already check for None here. Feel free to disable it in the pylintrc.

Reply:
disabled with a comment to match other code